schematic/store/synapse.py

```diff
@@ -182,36 +186,30 @@ def __init__(
         self.project_scope = project_scope
         self.storageFileview = CONFIG.synapse_master_fileview_id
         self.manifest = CONFIG.synapse_manifest_basename
+        self.root_synapse_cache = "/root/.synapseCache"
         self._query_fileview()
 
-    def _purge_synapse_cache(self, root_dir: str = "/var/www/.synapseCache/", maximum_storage_allowed_cache_gb=7):
+    def _purge_synapse_cache(self, maximum_storage_allowed_cache_gb=1):
         """
-        Purge synapse cache if it exceeds 7GB
+        Purge synapse cache if it exceeds 1GB
```

Comment:
purge synapse cache if it exceeds 1GB.

Reply:
Nit: update the doc.  this function is purging synapse cache when it exceeds a certain hard drive cap. The default is 1gb but it can be changed.

Reply:
Good point. Thanks Tom